.github/workflows/cicd.yml

```
@@ -0,0 +1,161 @@
+      - name: GitHub 인증
+        run: |
+          echo "${{ secrets.PERSONAL_ACCESS_TOKEN }}" | gh auth login --with-token
+
+      - name: 릴리즈 브랜치 생성
+        run: |
+          git checkout -b release/${{ github.run_id }}
+          git push origin release/${{ github.run_id }}
+
+      - name: 릴리즈 PR 생성
+        run: |
+          gh pr create --base main --head release/${{ github.run_id }} --title "release/${{ github.run_id }} -> main" --body "release pr"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)